### PR TITLE
Fix annotations

### LINKED ISSUE #18: Define new image tag annotations
Find agreement on new additional imagestream annotations:

| Status | Annotation key | Values | Description |
| --- | --- | --- | --- |
| Preexisting | `opendatahub.io/notebook-image-desc` | string | User facing image description |
| Proposed new | `opendatahub.io/notebook-image-message`| string encoded JSON | Validation output (errors and recommendations) |
| Preexisting | `opendatahub.io/notebook-image-name` | string | User facing image name |
| Proposed new, questionable | `opendatahub.io/notebook-image-origin` | string | Defines if the image is part of ODH distribution, admin imported or user created. Proposed values `Default`, `Admin` and `Custom` |
| Proposed new | `opendatahub.io/notebook-image-phase` | string | State of validation process: `Validating`, `Success`, `Failed` |
| Preexisting | `opendatahub.io/notebook-image-url` | url |  Container image url e.g. `quay.io/org/image:tag` |
| Proposed new | `opendatahub.io/notebook-image-visible` | boolean string | Decides if the imagestream is shown to users in spawner UI. Either `"true"` or `"false"` |

### LINKED ISSUE #30: Make pipelines queriable by ODH Dashboard API
Kubernetes supports native filters by labels when listing resources. In order to simplify parsing the relevant imagestreams and pipelineruns in ODH dashboard backend, let's enrich the pipeline resource and generated imagestream with additional label `app.kubernetes.io/created-by: byon`

### PR DESCRIPTION
**Pull request by tumido**
Depends on #34
Resolves: https://github.com/open-services-group/byon/issues/18
Resolves: https://github.com/open-services-group/byon/issues/30

### CODE REVIEW

File: charts/meteor-pipelines/templates/byon-validate-jupyterhub-image.yaml [hunk 1]
```diff
@@ -10,12 +10,12 @@ spec:
...
   steps:
-    - name: jupyter-book-build
+    - name: get-package-versions
       image: $(params.url)
       script: |
         #!/usr/bin/env bash
-        # Check Python versionand
-        echo  "Validating image:"
+        # Check Python version
+        echo  "Analyzing image:"
```
Comment (codificat) [inline charts/meteor-pipelines/templates/byon-validate-jupyterhub-image.yaml]: The changes to the Task  come from https://github.com/thoth-station/helm-charts/pull/34, right?

I believe this file was included in this PR by mistake?

Comment (tumido) [reply charts/meteor-pipelines/templates/byon-validate-jupyterhub-image.yaml]: No, this PR depends on #34, hence it includes the commit from the #34 PR. Once #34 is merged, this PR will become a single commit PR ->  https://github.com/thoth-station/helm-charts/pull/35/commits/41b75abfa58ea1486696255051a23e91160aadbe

**Pull request merged**